### DISCUSSION TITLE
Speaker classification from scratch

### DISCUSSION THREAD

**Discussion by Shrijoy-Chowdhury** [speechbrain/speechbrain#1657] [Q&A]
Hi I am new to speaker classification and I want to train a model from scratch using audio and speaker id that are present in csv file. I tried to change the parse_csv function but was not able to do so. Can anyone help me out here to understand how can I pull the data from csv file and then train a model.

**Comment by Shrijoy-Chowdhury** [comment]
[36448_train (1).csv](https://github.com/speechbrain/speechbrain/files/8103726/36448_train.1.csv)
This is the dataset

**Comment by TParcollet** [comment]
Hi, did you try to follow this tutorial: https://colab.research.google.com/drive/1UwisnAjr8nQF3UnrkIJ4abBMAWzVwBMh?usp=sharing ?

**Comment by Shrijoy-Chowdhury** [comment]
Hi, thank you for the prompt reply. I went through the tutorial and it was helpful. But there is a concern. In this tutorial the same audio has been split into train, validation and test, I want to train a model on some audios and later use the trained model to predict on another set of audios. Can you help me out with this? Waiting for your esteemed response.
